src/WebJobs.Script.WebHost/Controllers/InstanceController.cs

```diff
@@ -35,11 +36,35 @@ public InstanceController(IEnvironment environment, IInstanceManager instanceMan
         [HttpPost]
         [Route("admin/instance/assign")]
         [Authorize(Policy = PolicyNames.AdminAuthLevel)]
-        public async Task<IActionResult> Assign([FromBody] EncryptedHostAssignmentContext encryptedAssignmentContext)
+        public async Task<IActionResult> Assign([FromBody] HostAssignmentRequest hostAssignmentRequest)
         {
```

Comment:
Should we wrap this entire logic inside `IMetricsLogger.LatencyEvent`? @safihamid